ros/eloquent/ubuntu/bionic/images.yaml.em

```diff
@@ -43,10 +43,3 @@ images:
             - ros-comm
             - roscpp-tutorials
             - rospy-tutorials
-        downstream_packages:
```

Comment:
Is this not necessary anymore ?

Reply:
I looked at the ros_ci.yaml to remember to check which distro's where patched, but only saw noetic being treated as special. Patch re-applied here: https://github.com/osrf/docker_images/pull/479